### PR TITLE
Adding programming resources

### PR DESCRIPTION
**Pull request by nikita-jain-01**

### CODE REVIEW

File: programming-language-resources.md [hunk 1]
```diff
@@ -63,10 +63,12 @@
     - [Quick HTML Tutorial for Beginners](https://www.youtube.com/playlist?list=PLr6-GrHUlVf_ZNmuQSXdS197Oyr1L9sPB)
     - [HTML Crash Course for Beginners (1 hr)](https://www.youtube.com/watch?v=UB1O30fR-EE)
     - [Basic HTML and HTML5](https://www.freecodecamp.org/learn/responsive-web-design/basic-html-and-html5/)
+    - [W3 Schools](https://www.w3schools.com/html/default.asp)
```
Comment (sziolkow) [inline programming-language-resources.md:66]: The url can be shorter: https://www.w3schools.com/html/

File: programming-language-resources.md [hunk 2]
```diff
@@ -63,10 +63,12 @@
...
 - CSS
     - [Quick CSS Tutorial for Beginners](https://www.youtube.com/playlist?list=PLr6-GrHUlVf8JIgLcu3sHigvQjTw_aC9C)
     - [CSS Crash Course for absolute Beginners](https://www.youtube.com/watch?v=yfoY53QXEnI)
     - [Basic CSS](https://www.freecodecamp.org/learn/responsive-web-design/basic-css/)
+    - [W3 Schools](https://www.w3schools.com/css/default.asp)
```
Comment (sziolkow) [inline programming-language-resources.md:71]: Same as above. The url doesn't need dafault.asp

**Review by sziolkow — CHANGES_REQUESTED** [review]
Urls can be shorter

**Comment by jwasham** [comment]
Thanks!

**Pull request merged**